ui/tests/e2e/test.js

```diff
@@ -148,10 +158,29 @@ describe("End-to-end", () => {
 
     // Assert Relationship facet is added
     await waitForTextFacet("Relationship");
-    await assertFacet("Relationship", "1", "mother", "831");
+    await assertTextFacet("Relationship", "1", "mother", "831");
 
     // Make sure facet value is selected
-    await assertFacetValueSelected("Relationship", "paternal grandmother");
+    await assertTextFacetValueSelected("Relationship", "paternal grandmother");
+  });
+
+  test("[HistogramFacet] Participant facet", async () => {
+    // Assert facet rendered correctly
+    await assertHistogramFacet("Super Population", "3500", "African", "1018");
+
+    // Click on facet value
+    let facetBar = await getHistogramFacetBar("Super Population", "African");
+    await facetBar.click("input");
+    await waitForFacetsUpdate(1018);
+
+    // // Assert page updated correctly.
+    await assertHeaderTotalCount("1018");
+    await assertHistogramFacet("Gender", "1018", "male", "518");
+
+    // // Make sure non-selected facet values are gray.
+    facetBar = await getHistogramFacetBar("Super Population", "European");
+    const barColor = await page.evaluate(bar => bar.style.fill, facetBar);
+    expect(barColor).toBe("rgb(204, 207, 212)")
```

Comment:
Change to hex to match css

Reply:
Vega translates the hex to RGB so we have to validate this way, added a comment explaining.